src/Publisher.php

```diff
@@ -87,21 +87,6 @@ private function encode($key, $value): string
         return sprintf('%s=%s', $key, urlencode($value));
     }
 
-    private function publish(string $url, string $jwt, string $postData): string
```

Comment:
It is not unused: https://github.com/symfony/mercure/blob/96125742d4b6527ea5268212f00f797af84793fb/src/Publisher.php#L39

Reply:
euuh i see ^^